references/cli/addon.go

```diff
@@ -255,6 +288,10 @@ func NewAddonStatusCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 	}
 }
 
+func parseAddonArgsFromFlag(cmd *cobra.Command) {
+	cmd.Flags().StringP(addonLocalDir, "f", "", "enable addon with a local dir path")
```

Comment:
```suggestion
	cmd.Flags().StringP(addonLocalDir, "d", "", "enable addon with a local dir path")
```
Is `-d` or `-l` better?

Reply:
yes, it's not a file

Reply:
the best user expericnce is helm like:


```
vela addon enable <addonname>
vela addon enable <local-dir>
```

The args can be local dir or name, you can check it in code.

Reply:
fixed

Reply:
> the best user expericnce is helm like:
> 
> ```
> vela addon enable <addonname>
> vela addon enable <local-dir>
> ```
> 
> The args can be local dir or name, you can check it in code.

What if there is conflict? IMO a flag is needed.